Fixtures/Miscellaneous/ModuleAliasing/DirectDeps/AppPkg/Package.swift

```diff
@@ -10,14 +10,14 @@ let package = Package(
         .watchOS(.v5)
     ],
     dependencies: [
-        .package(url: "../GamePkg", from: "1.0.0"),
+        .package(path: "../GamePkg"),
     ],
     targets: [
         .executableTarget(
             name: "App",
             dependencies: [
                 "Utils",
-                .product(name: "Utils",
+                .product(name: "UtilsProd",
```

Comment:
@elsh its very common that folks use the same name for the target and the product. is this causing issues?